rtp-session.go

```diff
@@ -81,6 +81,20 @@ func (s *RTPSession) Decode(packetData []byte) (*RTPPacket, error) {
 	return p, nil
 }
 
+func (s *RTPSession) DecodeRTCP(packetData []byte) (*RTCPCompoundPacket, error) {
+
+	p, err := NewSRTCPPacket(packetData)
+	if err != nil {
+		return nil, err
+	}
+
+	if s.cipher != NONE {
+		return p.DecryptGCM(s.key, s.salt)
```

Comment:
This is incorrect.  You need to derive a separate SRTCP key and salt, and store them in fresh fields on the session object.

https://tools.ietf.org/html/rfc3711#section-4.3.2